x/ccv/provider/keeper/staking_keeper_interface.go

```diff
@@ -0,0 +1,80 @@
+package keeper
+
+import (
+	"context"
+
+	"cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+)
+
+// IterateBondedValidatorsByPower iterates over the consensus-active validators by power.
+// The same as IterateBondedValidatorsByPower in the StakingKeeper,
+// but only returns the first MaxProviderConsensusValidators validators.
+// This is used to implement the interface of the staking keeper to interface with
```

Comment:
Not sure, but "to interact with" might sound slightly better than "to interface with"